lib/nfg_ui/bootstrap/utilities/modalable.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module NfgUi
+  module Bootstrap
+    module Utilities
+      # Delivers the dismissibility of the component to the HAML partial
```

Comment:
@jonathanroehm Is this comment true, that the purpose of this module is to make a modal dismissable?

Reply:
hah! Thanks for spotting that.